### PR TITLE
fix: Remove broken invidious embeds

### LINKED ISSUE #2564: Invidious video embeds are broken
### Bug description

Some (maybe all) Invidious video embeds seem to be broken.

### Affected pages

- https://www.privacyguides.org/en/os/linux-overview/#release-cycle

### Browser

Firefox

### Operating System

Linux

### Before submitting

- [X] I am reporting something that is broken on the website, not making a suggestion.
- [X] I agree to the [Community Code of Conduct](https://www.privacyguides.org/coc).

### PR DESCRIPTION
**Pull request by kimg45**
Changes proposed in this PR:

- remove broken invidious embeds and replace with YouTube links

One of the videos doesn’t even seem to be available anymore. I felt the best way to simplify and prevent future breakage is to just link to the original YouTube video instead of embedding it.

Closes #2564 

<summary>

- [x] I agree to the terms listed below:
  <details><summary>Contribution terms (click to expand)</summary>
  1) I am the sole author of this work.
  2) I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
  3) I have disclosed any relevant conflicts of interest in my post.
  4) I agree to the Community Code of Conduct.
  </details>

### CODE REVIEW

**Comment by dngray** [comment]
We should probably investigate to see if these can be fixed first before reverting.

**Review by jonaharagon — APPROVED** [review]

**Comment by jonaharagon** [comment]
I don't think this can be fixed long-term, following Google's changes to YouTube around July 7th.

File: docs/basics/multi-factor-authentication.md [hunk 1]
```diff
@@ -75,11 +75,7 @@ WebAuthn is the most secure and private form of second factor authentication. Wh
 
 When you create an account, the public key is sent to the service, then when you log in, the service will require you to "sign" some data with your private key. The benefit of this is that no password data is ever stored by the service, so there is nothing for an adversary to steal.
 
-This presentation discusses the history of password authentication, the pitfalls (such as password reuse), and discussion of FIDO2 and [WebAuthn](https://webauthn.guide) standards.
-
-<div class="yt-embed">
-  <iframe width="560" height="315" src="https://invidious.privacyguides.net/embed/aMo4ZlWznao?local=true" title="How FIDO2 and WebAuthn Stop Account Takeovers" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-</div>
+This [presentation](https://youtu.be/aMo4ZlWznao) discusses the history of password authentication, the pitfalls (such as password reuse), and discussion of FIDO2 and [WebAuthn](https://webauthn.guide) standards.
```
Comment (redoomed1) [inline docs/basics/multi-factor-authentication.md:78]:
```suggestion
This presentation discusses the history of password authentication, the pitfalls (such as password reuse), and the standards for FIDO2 and [WebAuthn](https://webauthn.guide):

- [How FIDO2 and WebAuthn Stop Account Takeovers](https://youtu.be/aMo4ZlWznao) <small>(YouTube)</small>
```

File: docs/os/linux-overview.md [hunk 2]
```diff
@@ -43,11 +43,7 @@ We highly recommend that you choose distributions which stay close to the stable
...
 
-We don’t believe holding packages back and applying interim patches is a good idea, as it diverges from the way the developer might have intended the software to work. [Richard Brown](https://rootco.de/aboutme) has a presentation about this:
-
-<div class="yt-embed">
-  <iframe width="560" height="315" src="https://invidious.privacyguides.net/embed/i8c0mg_mS7U?local=true" title="Regular Releases are Wrong, Roll for your life" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-</div>
+We don’t believe holding packages back and applying interim patches is a good idea, as it diverges from the way the developer might have intended the software to work.
```
Comment (redoomed1) [inline docs/os/linux-overview.md:46]:
```suggestion
We don’t believe holding packages back and applying interim patches is a good idea, as it diverges from the way the developer might have intended the software to work. [Richard Brown](https://rootco.de/aboutme) has a presentation about this:

- [Regular Releases are Wrong, Roll for your life](https://youtu.be/i8c0mg_mS7U) <small>(YouTube)</small>
```

File: docs/os/linux-overview.md [hunk 3]
```diff
@@ -57,11 +53,7 @@ Atomic updating distributions apply updates in full or not at all. Typically, tr
...
 A transactional update system creates a snapshot that is made before and after an update is applied. If an update fails at any time (perhaps due to a power failure), the update can be easily rolled back to a “last known good state."
 
-The Atomic update method is used for [distributions](../desktop.md#atomic-distributions) like Silverblue, Tumbleweed, and NixOS and can achieve reliability with this model. [Adam Šamalík](https://twitter.com/adsamalik) provided a presentation on how `rpm-ostree` works with Silverblue:
-
-<div class="yt-embed">
-  <iframe width="560" height="315" src="https://invidious.privacyguides.net/embed/-hpV5l-gJnQ?local=true" title="Let's try Fedora Silverblue — an immutable desktop OS! - Adam Šamalik" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-</div>
+The Atomic update method is used for [distributions](../desktop.md#atomic-distributions) like Silverblue, Tumbleweed, and NixOS and can achieve reliability with this model. [Adam Šamalík](https://twitter.com/adsamalik) provided a [presentation](https://youtu.be/-hpV5l-gJnQ) on how `rpm-ostree` works with Silverblue.
```
Comment (redoomed1) [inline docs/os/linux-overview.md:56]:
```suggestion
The Atomic update method is used for [distributions](../desktop.md#atomic-distributions) like Silverblue, Tumbleweed, and NixOS and can achieve reliability with this model. [Adam Šamalík](https://twitter.com/adsamalik) provided a [presentation](https://youtu.be/-hpV5l-gJnQ) on how `rpm-ostree` works with Silverblue.

- [Let's try Fedora Silverblue — an immutable desktop OS! - Adam Šamalik](https://youtu.be/aMo4ZlWznao) <small>(YouTube)</small>
```

**Review by redoomed1 — CHANGES_REQUESTED** [review]
I think the presentation of the YouTube links here should be consistent with the way they're presented here: https://www.privacyguides.org/en/advanced/tor-overview/#additional-resources

There are also advantages of signaling that these are YouTube videos rather than embedding them in-text:

1. Indicating the link is a YouTube link avoids surprising readers in case they don't mouse over the link for the browser tooltip or long-press on the link on mobile
2. As a corollary, it gives readers the freedom of using a YouTube frontend once they see that it's a YouTube link.

**Review by redoomed1 — APPROVED** [review]
LGTM

(Also, sorry for the [merge commit](https://github.com/privacyguides/privacyguides.org/pull/2585/commits/726a9c0c7f8a47fb4f59add4b9b7e18047113a92), I should've rebased first to avoid a merge conflict :pensive: )

**Pull request merged**